src/main/kotlin/racingCar/controller/RacingCarController.kt

```diff
@@ -0,0 +1,28 @@
+package racingCar.controller
+
+import racingCar.domain.Cars
+import racingCar.domain.Car
+import racingCar.view.InputView
+import racingCar.view.OutputView
+import java.util.Arrays
+
+fun main() {
+    OutputView.printCarCount()
+    val carCount = InputView.inputNumber()
+    OutputView.printTryCount()
+    val tryCount = InputView.inputNumber()
+    OutputView.printExecuteResult()
+
+    // carList 초기화
+    val carList: MutableList<Car> = ArrayList()
+    for (i in 0 until carCount) {
+        val car = Car()
+        carList.add(car)
+    }
```

Comment:
[List factory 함수](https://kotlinlang.org/docs/constructing-collections.html#initializer-functions-for-lists)를 사용하면 Mutable 타입을 선언하지 않아도 됩니다.
```suggestion
    val cars: List<Car> = List(carList) { Car() }
```

이런 종류의 함수를 '[가짜 생성자](https://blog.kotlin-academy.com/item-30-consider-factory-functions-instead-of-constructors-e1c747fc475)'라고도 부릅니다 :slightly_smiling_face: